agent/handlers/task_server_setup_test.go

```diff
@@ -1675,6 +1685,57 @@ func TestV4ContainerAssociation(t *testing.T) {
 	assert.Equal(t, expectedAssociationResponse, string(res))
 }
 
+func TestGQLMetadata(t *testing.T) {
+	testCases := []struct {
+		name                 string
+		gqlCreateSchemaError error
```

Comment:
Probably can include the expected `recorder.Code` and the expected `FailedGQLResponse` here if more test cases is going to be added later 😃